kernel/kernel-config-server/pom.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>io.mosip.kernel</groupId>
```

Comment:
Yes. It is the same and now migrated to java21.

Reply:
As discussed, we have to explore the below prebuilt images instead of us maintaining.
https://hub.docker.com/r/hyness/spring-cloud-config-server/tags
https://hub.docker.com/r/springcloud/configserver/tags
Marking resolved for now since we will create a task to explore this.